jax/interpreters/pxla.py

```diff
@@ -1200,7 +1176,13 @@ def partitioned_sharding_spec(num_partitions: int,
 
 def execute_replicated(compiled, backend, in_handler, out_handler, *args):
   input_bufs = in_handler(args)
-  out_bufs = compiled.execute_on_local_devices(list(input_bufs))
+  try:
```

Comment:
FYI I bumped the minimum jaxlib version to 0.1.60, released just now, so you don't need this workaround any more.

Reply:
Oh, sweet! Thanks.

Reply:
Aww, bummer. Looks like it isn't quite ready yet. I'm getting errors.

Reply:
You were too quick. The jaxlib version increase isn't submitted yet...